app/src/main/res/values/edit_name_list_strings.xml

```diff
@@ -27,7 +27,7 @@
     <string name="import_names_from_list">Import names from another list(s)</string>
 
     <string name="confirm_deletion_title">Confirm List Deletion</string>
-    <string name="confirm_deletion_message">Are you sure you want to delete this name list?</string>
+    <string name="confirm_deletion_message">Are you sure you want to delete your name list</string>
```

Comment:
This makes the code a little longer, but I think this is better separation of responsibilities. We want to put as much of the text into the XML as possible.

```suggestion
    <string name="confirm_deletion_message">Are you sure you want to delete your name list %s?</string>
```